products/compute/api.yaml

```diff
@@ -669,6 +670,26 @@ objects:
         name: 'lastDetachTimestamp'
         description: 'Last dettach timestamp in RFC3339 text format.'
         output: true
+      - !ruby/object:Api::Type::NameValues
+        name: 'labels'
+        description: |
+          Labels to apply to this disk. These can be later modified by the setLabels method.
+          An object containing a list of key->value pairs.
+        key_type: Api::Type::String
+        value_type: Api::Type::String
+        update_verb: :POST
+        update_url: 'projects/{{project}}/zones/{{zone}}/disks/{{name}}/setLabels'
+      - !ruby/object:Api::Type::String
+        name: 'labelFingerprint'
+        description: |
+          A fingerprint for the labels being applied to this disk, which is essentially a
+          hash of the labels set used for optimistic locking. The fingerprint is initially
+          generated by Compute Engine and changes after every request to modify or update
+          labels. You must always provide an up-to-date fingerprint hash in order to
+          update or change labels.
+        update_verb: :POST
```

Comment:
Can you add a comment here to explain that it's not actually the fingerprint you're trying to update, but that you're providing the fingerprint when updating the label so they have the same update url? Otherwise it could be a bit confusing to have an output attribute that can be updated.